modelator/const_values.py

```diff
@@ -39,6 +39,10 @@ def apalache_release_url(expected_version):
     return f"https://github.com/informalsystems/apalache/releases/download/v{expected_version}/apalache.zip"
 
 
+def apalache_checksum_url(expected_version: str):
```

Comment:
Does this function belongs here or in `apalache_jar.py`?

Reply:
I added it here because `apalache_release_url` is here.